packages/solutions/README.md

```diff
@@ -17,10 +17,71 @@
 npm install @esri/hub-solutions
 ```
 
+## Coarse Grained API
```

Comment:
README.md > guide > john 😄

Reply:
Once we actually *have* solutions code, we can move this... right now it's more a "plan" for the API we want...